File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.js
function solution(binomial) {
    let result = binomial.split(' ')
    if(result[1] == '+') return Number(result[0]) + Number(result[2])
    if(result[1] == '-') return Number(result[0]) - Number(result[2])
    if(result[1] == '*') return Number(result[0]) * Number(result[2])
    if(result[1] == '/') return Number(result[0]) / Number(result[2])
    
}

File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.py
def solution(binomial):
    split = binomial.split(' ')
    if split[1] == '+':
        return int(split[0]) + int(split[2])
    elif split[1] == '-':
        return int(split[0]) - int(split[2])
    elif split[1] == '*':
        return int(split[0]) * int(split[2])